.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Description
+
+Please explain the changes you made, including a reference to the related issue if applicable
+
+### Checklist
+- [ ] Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
```

Comment:
does clang-format detect trailing-whitespace errors? If not, that could be another item.